.gitignore

```diff
@@ -14,3 +14,4 @@
 /default.etcd
 *.pyc
 __pycache__
+management-state/
```

Comment:
can you prefix with `/` otherwise it applies to any subfolder named that too (which isn't too common but really sucks when you hit the issues of random stuff getting ignored)